### PR TITLE
Variable scope, closure

### PR DESCRIPTION
**Pull request by mahdiHashemi14**
Please review and comment / commit if anything is wrong

### CODE REVIEW

File: 1-js/06-advanced-functions/03-closure/1-closure-latest-changes/task.md [hunk 1]
```diff
@@ -15,9 +15,8 @@ function sayHi() {
...
 
-sayHi(); // what will it show: "John" or "Pete"?
+sayHi(); // ؟"Pete" یا "John" :چه چیزی نمایش خواهد داد
 ```
+چنین موقعیت‌هایی در توسعه هم سمت مرورگر و هم سمت سرور رایج هستند. ممکن است یک تابع قرار باشد بعد از اینکه ساخته شد اجرا شود، برای مثال بعد از اینکه کاربر کاری انجام داد یا بعد از یک درخواست شبکه.
 
-Such situations are common both in browser and server-side development. A function may be scheduled to execute later than it is created, for instance after a user action or a network request.
-
-So, the question is: does it pick up the latest changes?
+پس سوال این است: آیا این تابع آخرین تغییرات را دریافت می‌کند؟
```
Comment (mahdyar) [inline 1-js/06-advanced-functions/03-closure/1-closure-latest-changes/task.md:22]: Line numbers should be the same. (You deleted the new line in line 20)

File: 1-js/06-advanced-functions/03-closure/2-closure-variable-access/solution.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 The answer is: **Pete**.
+جواب: **Pete**.
```
Comment (mahdyar) [inline 1-js/06-advanced-functions/03-closure/2-closure-variable-access/solution.md:3]: You deleted the new line here as well.

**Review by mahdyar — CHANGES_REQUESTED** [review]

**Comment by mahdiHashemi14** [comment]
Thank you @mahdyar
/done

**Review by mahdyar — APPROVED** [review]
Thank you, @mahdiHashemi14!

**Pull request merged**